src/shared/replication/replication_rules.rs

```diff
@@ -1,51 +1,141 @@
-    /// marker component.
+    /// If present on an entity with [`Replicated`](super::Replicated) component,
+    /// it will be serialized and deserialized as-is using [`postcard`]
+    /// and sent at [`SendRate::EveryTick`]. To customize this, use [`Self::replicate_with`].
     ///
-    /// Component will be serialized and deserialized as-is using postcard.
-    /// To customize it, use [`Self::replicate_group`].
-    ///
-    /// See also [`Self::replicate_with`] and the section on [`components`](../../index.html#components)
-    /// from the quick start guide.
+    /// See also the section on [`components`](../../index.html#components) from the quick start guide.
     fn replicate<C>(&mut self) -> &mut Self
     where
         C: Component<Mutability: MutWrite<C>> + Serialize + DeserializeOwned,
     {
-        self.replicate_with::<C>(RuleFns::default())
+        self.replicate_with(RuleFns::<C>::default())
+    }
+
+    /// Sames as [`Self::replicate`], but uses [`SendRate::Once`].
+    fn replicate_once<C>(&mut self) -> &mut Self
+    where
+        C: Component<Mutability: MutWrite<C>> + Serialize + DeserializeOwned,
+    {
+        self.replicate_with((RuleFns::<C>::default(), SendRate::Once))
     }
 
-    #[deprecated(note = "no longer needed, just use `replicate` instead")]
-    fn replicate_mapped<C>(&mut self) -> &mut Self
+    /// Sames as [`Self::replicate`], but uses [`SendRate::Periodic`] with the given period.
+    fn replicate_periodic<C>(&mut self, period: u32) -> &mut Self
     where
-        C: Component<Mutability: MutWrite<C>> + Serialize + DeserializeOwned + MapEntities,
+        C: Component<Mutability: MutWrite<C>> + Serialize + DeserializeOwned,
     {
-        self.replicate::<C>()
+        self.replicate_with((RuleFns::<C>::default(), SendRate::Periodic(period)))
     }
 
     /**
-    Same as [`Self::replicate`], but uses the specified functions for serialization and deserialization.
+    Defines a [`ReplicationRule`] for a bundle.
+
+    Implemented for tuples of components. Use it to conveniently create a rule with
+    default ser/de functions and [`SendRate::EveryTick`] for all components.
+    To customize this, use [`Self::replicate_with`].
+
+    Can also be implemented manually for user-defined bundles.
+
+    # Examples
+
+    ```
+    use bevy::prelude::*;
+    use bevy_replicon::{
+        bytes::Bytes,
+        shared::replication::{
+            replication_registry::{
+                ctx::{SerializeCtx, WriteCtx},
+                ReplicationRegistry,
+            },
+            replication_rules::{ReplicationBundle, ReplicationRule, ComponentRule},
+        },
+        prelude::*,
+    };
+    use serde::{Deserialize, Serialize};
+
+    # let mut app = App::new();
+    # app.add_plugins(RepliconPlugins);
+    app.replicate_bundle::<(Name, City)>() // Tuple of components is also a bundle!
+        .replicate_bundle::<PlayerBundle>();
```

Comment:
`PlayerBundle` includes `Replicated`, but that component doesn't need to be replicated.

Reply:
Good point, removed it from the bundle.